test/mocha.opts

```diff
@@ -1,4 +1,4 @@
 --ui tdd
 --timeout 240s
 --reporter spec
---require source-map-support/register
+#--require source-map-support/register
```

Comment:
I had a lot of trouble getting source-map-support to work for some reason, since it's not critical i decided to disable it for the time being.